src/slic3r/GUI/PrinterWebView.cpp

```diff
@@ -30,6 +30,8 @@ PrinterWebView::PrinterWebView(wxWindow *parent)
         return;
     }
 
+    Bind(wxEVT_WEBVIEW_ERROR, &PrinterWebView::OnError, this);
```

Comment:
Binding to the error event and logging the details. on the webview was how I was able to confirm my suspicion that it was an OS security policy thing.